README.md

```diff
@@ -3,111 +3,111 @@
+| Stop | Stops dev mode. Liberty must be running in dev mode to use this command. |
+| Run tests | Runs the unit tests and integration tests that are configured for your project. Liberty must be running in dev mode to use this command. |
+| View integration test report (Maven) | Views the integration test report file it exists at `/target/site/failsafe-report.html`. |
+| View unit test report (Maven) | Views the unit test report file if it exists at `/target/site/surefire-report.html`. |
+| View test report (Gradle) | Opens the test report file, if it exists at the `build/reports/tests/test/index.html` default location. This action command is available only to Gradle projects. Gradle projects have only a single action command for test result reporting. |
 
-1. `git clone https://github.com/OpenLiberty/liberty-tools-vscode`
-2. `cd liberty-tools-vscode`
-3. Run `npm install`
-4. Run `npm run build`
-5. Run `npm run compile`
-6. Run the extension in Debug and Run mode by selecting `Run Extension` or `F5`
+## Configurable User Settings
 
-   Alternatively, build a `.vsix` file:
+| Setting | Description | Default Value |
+| --- | --- | --- |
+| liberty.terminal.useJavaHome | If this value is true, and if the setting `java.home` has a value, then the environment variable `JAVA_HOME` will be set to the value of `java.home` when a new terminal window is created. | False         |
+| xml.java.home | This property allows a user to define their LemMinX language server runtime without altering the `JAVA_HOME` environment variable.  | Not set |
 
-   - `vsce package` to generate the `liberty-tools-vscode-xxx.vsix` file
-   - Install the extension to Visual Studio Code by `View/Command Palette`
-   - Select `Extensions: Install from VSIX...` and choose the generated `liberty-tools-vscode-xxx.vsix` file
+### External Settings
 
-### Localization
+The following settings provided by external extensions will be honoured when executing dev mode commands.
 
-#### package.json
-This follows Visual Studio Code extension standard: add localized strings in `package.nls.{locale}.json`.
-The default nls message file is `package.nls.json`.
+| Setting | Description | Provided By |
+| --- | --- | --- |
+| maven.executable.path | Maven commands executed by dev mode will honour this setting. When this value is empty, it tries to use `mvn` or `mvnw` according to the value of `maven.executable.preferMavenWrapper`. | [Maven for Java extension](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-maven) |
+| maven.executable.preferMavenWrapper | Maven commands executed by dev mode will honour this setting. If true, it tries to use `mvnw` if a Maven wrapper file can be found. Otherwise it will use `mvn`. | [Maven for Java extension](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-maven) |
```

Comment:
```suggestion
| `maven.executable.preferMavenWrapper` | Maven commands executed by dev mode will honour this setting. If true, it tries to use `mvnw` if a Maven wrapper file can be found. Otherwise it will use `mvn`. | [Maven for Java extension](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-maven) |
```